src/app/pages/ArticlePage/index.test.tsx

```diff
@@ -74,12 +77,16 @@ const Context = ({
   showAdsBasedOnLocation = false,
   isApp = false,
   promo = null,
+  isAmp = false,
+  pathname = '/pathname',
```

Comment:
Could we use a pathname which is similar to a news/sport amp optimo article?

Reply:
Replaced `pathname` with `id` which doesn't have a default. Unsure if it should, though no other tests will use it so I haven't added one.